### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DeadManWalkingTO**
Quality metadata badges.
Build instructions: Usage of the latest version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,7 +157,7 @@ invokes cmake commands as needed.
 * Change to the root of the source code directory, change to the most recent release tag, and build:
 
         cd aeon
-        git checkout v0.12.2.1
+        git checkout v0.12.6.0
```
[the remark this answers is not in the document]
Comment (DeadManWalkingTO) [reply README.md]: Fixed

**Comment by stoffu** [comment]
Please squash these two commits to one.

File: README.md [hunk 2]
```diff
@@ -143,7 +157,7 @@ invokes cmake commands as needed.
 * Change to the root of the source code directory, change to the most recent release tag, and build:
 
         cd aeon
-        git checkout v0.12.2.1
+        git checkout v0.12.6.0-aeon
```
Comment (stoffu) [inline README.md:160]: Maybe could you please also update the number in the "Recommended Aeon version" column of the Aeon version table?

Comment (DeadManWalkingTO) [reply README.md:160]: Done!

**Review by stoffu — APPROVED** [review]
Thank you

**Pull request merged**